### PR TITLE
[add] Vendor Profile Component setup

### PR DESCRIPTION
**Pull request by edbella**
Screenshot 2020-01-03 at 23 01 01

### CODE REVIEW

**Comment by edbella** [comment]
@enyeInc @justiceotuya @olajohn-ajiboye Would appreciate a review here. Been up since 2 days ago and it seems you missed it.

**Comment by justiceotuya** [comment]
> @enyeInc @justiceotuya @olajohn-ajiboye Would appreciate a review here. Been up since 2 days ago and it seems you missed it.

sorry guy

File: src/components/vendorDashboard/components/profile/components/ProfileData.jsx [hunk 1]
```diff
@@ -1,32 +1,151 @@
...
+        <>
+            <Form.Item label="Email Address" colon={false}>
+                <Input type="text" value={email} disabled />
+            </Form.Item>
+            <Form.Item label="Your Name" colon={false}>
+                <Input type="text" value={displayName} />
+            </Form.Item>
+            <Form.Item label="Phone Number" colon={false}>
+                <Input type="text" placeholder={PHONE_NUMBER_PLACEHOLDER} value={phoneNumber} />
+            </Form.Item>
```
Comment (Way2nnadi) [inline src/components/vendorDashboard/components/profile/components/ProfileData.jsx]: use array mapping to creating these form items...come on you know this already

File: .firebase/hosting.YnVpbGQ.cache [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
+static/js/runtime-main.46d71886.js,1578046377969,792eada5e299ea146f24cbd2b8b9f467888cc356d931be2fc5d61417f24c8f7c
+static/js/runtime-main.46d71886.js.map,1578046378003,7ca32f7f2cbb6adf84ae72eb7077914e5fb336d65f69a5cefb88b1bcae17ca73
+static/media/placeholder-logo.271d8e5a.svg,1578046378003,d0008135c25b59821f7066c3c8810788c463abeca953f15a05fb4abe3e8de741
+logo512.png,1578046336933,55a8a8fc728e1bf5023516482fec7779c6f543625baa88eefc4cf6f784221957
+static/media/hero-illustration.fd56dee5.svg,1578046377964,322f52523ae802d4e288dc6b8ee7778f133582ad1a4201c9e233258230804370
+static/js/main.5cff87ba.chunk.js.map,1578046378006,d1185d352f134f540fbb2cf9dc620587b85d2a524a14515691e238b6ae21ec74
+static/css/2.03cd2798.chunk.css,1578046378005,b2e33ba416d3e9b1d310655743263238a91ec816a5cbf73c2fab57d067f83ca5
+static/css/2.03cd2798.chunk.css.map,1578046378006,dc1a9c384ec817a1db9536321984c6d740ca52c695040a3434af9b94dd36dac2
+static/js/2.a44dbec4.chunk.js,1578046378003,5146105937ce80a79072558e72c9f5d04818914ce015cce1c3bc87235cad459b
+static/js/2.a44dbec4.chunk.js.map,1578046378005,dc6e703fe98c1bc2fbbd423f2702441a658cd7a8039057847065eb5b203e2036
```
Comment (Way2nnadi) [inline .firebase/hosting.YnVpbGQ.cache]: Why are you committing this file?

File: src/components/vendorDashboard/components/profile/components/ProfileData.jsx (hunk 1 above)
Comment (edbella) [reply src/components/vendorDashboard/components/profile/components/ProfileData.jsx]: @enyeInc @Way2nnadi Let me just take it out then. If I do that now, the changes will shoot beyond the 250 target. That's why I left it. Was going to turn it into an array mapping in the next PR.

File: .firebase/hosting.YnVpbGQ.cache (hunk 2 above)
Comment (edbella) [reply .firebase/hosting.YnVpbGQ.cache]: Alright.

**Pull request merged**